pkg/tests/api/plugins/data/expectedListResp.json

```diff
@@ -1440,6 +1440,42 @@
     "signatureType": "",
     "signatureOrg": ""
   },
+  {
+    "name": "Traces",
+    "type": "panel",
+    "id": "traces",
+    "enabled": true,
+    "pinned": false,
+    "info": {
+      "author": {
+        "name": "Grafana Labs",
+        "url": "https://grafana.com"
+      },
+      "description": "",
+      "links": null,
+      "logos": {
+        "small": "public/app/plugins/panel/traces/img/icn-singlestat-panel.svg",
```

Comment:
Nit: Would probably just update this to be its own specific named logo as it looks like a typo at first 🙈

Reply:
100%, we're just about decided on what logo to use :) https://raintank-corp.slack.com/archives/C02U5AAVAES/p1651072230103229